src/components/Table.jsx

```diff
@@ -0,0 +1,94 @@
+import { Button } from '@mui/material';
+import { DataGrid } from '@mui/x-data-grid';
+import data from './data.json';
+
+const multiplePublishButtons = (params) => {
```

Comment:
Move this to another component and call it `PlatformActions`.
**Please ensure that all components name are starting with Capital Letter!**